client_sdk/avalon_client_sdk/ethereum/ethereum_worker_registry_list_client.py

```diff
@@ -144,6 +149,19 @@ def registry_set_status(self, org_id, status):
                 "direct registry contract instance is not initialized")
 
     def registry_lookup(self, app_type_id=None):
+        return self.__registry_lookup(app_type_id, None, False)
+
+    def registry_lookup_next(self, app_type_id, lookup_tag):
+        return self.__registry_lookup(app_type_id, lookup_tag, True)
+
+    def __registry_lookup(self, app_type_id=None, lookup_tag=None, lookup_next=False):
+        """
+        This function handles both the lookup & lookup_next calls. It looks
+        for the flag, lookup_next which if enabled, it would discard all
+        entries in result set till the look_up tag. It paginates results
+        post this ans returns as next page.
```

Comment:
s/post this ans returns as next page./after this and returns it as the next page.